src/manager/monitor.c

```diff
@@ -194,6 +195,84 @@ static int monitor_method_close(sd_bus_message *m, void *userdata, UNUSED sd_bus
  **************** org.containers.hirte.Monitor.Subscribe *****************
  ************************************************************************/
 
+static bool subscribe_on_node(Manager *manager, Monitor *monitor, const char *node, const char *unit) {
+        _cleanup_free_ char *unit_copy = NULL;
+        if (!copy_str(&unit_copy, unit)) {
+                return false;
+        }
+
+        char *saveptr = NULL;
+        char *unit_name = strtok_r(unit_copy, ",", &saveptr);
```

Comment:
I don't like using stringly typed things like this. If we want to pass multiple units, we should do it with actually typed dbus arrays.

Reply:
I totally agree. This has been removed in one of the subsequent commits. By the latest rebase I cleaned up that commit as well - its now in the `SubscribeList` API method.